leaves/text_input.mli

```diff
@@ -0,0 +1,58 @@
+type t = {
```

Comment:
**fix**. let's make this abstract.

Reply:
In this case, should we rename `value` -> `text` for `val make : ~value: string -> ...`?

Reply:
... alternatively simply remove the named argument for value

Reply:
I did the latter: [7f96876](https://github.com/leostera/minttea/pull/16/commits/7f96876cd5e96fa2efea3a79bc9492e94e192d65)